hoon/reference/auras.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "Auras"
+weight = 70
+template = "doc.html"
++++
+
+```
```

Comment:
A brief bit of copy explaining what auras are would probably be nice. A backlink to the hoon school explanation would be a good idea too.